1. values added:  20
2. final result: 20
3. values added:  20
4. The code returns an error. let is only accessible within the block where it is defined, and line 13 attempts to reference it outside of this scope.
5. The code returns an error. const cannot have a value reassigned to it after it is defined, so the process exits on line 7 and never reaches line 9.
6. The code returns an error. const cannot have a value reassigned to it after it is defined, so the process exits on line 7 and never reaches line 13.